streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -529,14 +530,23 @@ public class StreamsConfig extends AbstractConfig {
     private static final String DEFAULT_PRODUCTION_EXCEPTION_HANDLER_CLASS_DOC = "Exception handling class that implements the <code>org.apache.kafka.streams.errors.ProductionExceptionHandler</code> interface.";
 
     /** {@code default.dsl.store} */
+    @Deprecated
     @SuppressWarnings("WeakerAccess")
     public static final String DEFAULT_DSL_STORE_CONFIG = "default.dsl.store";
+    @Deprecated
     public static final String DEFAULT_DSL_STORE_DOC = "The default state store type used by DSL operators.";
 
+    @Deprecated
     public static final String ROCKS_DB = "rocksDB";
+    @Deprecated
     public static final String IN_MEMORY = "in_memory";
     public static final String DEFAULT_DSL_STORE = ROCKS_DB;
```

Comment:
This should also be deprecated, right? (It's not used by the new config code I take it?)